src/Service/TokenService.cs

```diff
@@ -92,9 +92,9 @@ public T DecodeToken<T>(string token, string prefix, bool contractless = false)
             }
             else
             {
-                _log.LogWarning("Could not remove prefix={prefix}", prefix);
-                // todo: Uncomment to require new token format. Should be safe to do after 2020-01-30:
-                throw new ApiException($"The token you sent was not correct. The token used for this endpoint should start with '{prefix}'. Make sure you are not sending the wrong value.", 400);
+                var invalidInput = token[..Math.Min(10, token.Length)];
+                _log.LogWarning("Could not remove prefix={prefix}, token started with {InvalidInput}", prefix, invalidInput);

```

Comment:
nit: That we are mixing camelCase and PascalCase for the named holes but I know we this isn't the only place and part of why I might turn on this rule soon: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1727

Reply:
I agree that we should turn on that rule. @justindbaur